OpenRA.Game/WAngle.cs

```diff
@@ -41,6 +41,8 @@ public WAngle(int a)
 		public bool Equals(WAngle other) { return other == this; }
 		public override bool Equals(object obj) { return obj is WAngle && Equals((WAngle)obj); }
 
+		public int Facing { get { return Angle / 4; } }
```

Comment:
These are essentially fundamental properties of the owning type (like `X`, `Y`, `Angle`, etc), and not a conversion to another complex object.